yaml-tests/src/main/java/com/apple/foundationdb/relational/yamltests/configs/MultiServerConfig.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * MultiServerConfig.java
+ *
+ * This source file is part of the FoundationDB open source project
+ *
+ * Copyright 2015-2025 Apple Inc. and the FoundationDB project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.apple.foundationdb.relational.yamltests.configs;
+
+import com.apple.foundationdb.relational.yamltests.MultiServerConnectionFactory;
+import com.apple.foundationdb.relational.yamltests.YamlRunner;
+import com.apple.foundationdb.relational.yamltests.server.ExternalServer;
+import org.junit.jupiter.api.Assumptions;
+
+import javax.annotation.Nonnull;
+import java.net.URI;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.List;
+import java.util.Set;
+
+/**
+ * Run against an embedded JDBC driver, and an external server, alternating commands that go against each.
+ */
+public class MultiServerConfig implements YamlTestConfig {
+
+    private final JDBCInProcessConfig embeddedConfig;
+    private final ExternalServer externalServer;
+    private int initialConnection;
+
+    public MultiServerConfig(final int initialConnection, final int grpcPort, final int httpPort) {
+        this.initialConnection = initialConnection;
+        embeddedConfig = new JDBCInProcessConfig();
```

Comment:
Nit: `this` looks like it could have extended `JDBCInProcessConfig`

Reply:
That being said, in order to get this to play well with `@ExcludeYamlTestConfig`, it was easier to change it to extend `JDBCInProcessConfig`